test/ocsp/checkocsp/checkocsp.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/letsencrypt/boulder/test/ocsp/helper"
+)
+
+func main() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, `
+OCSP-checking tool. Provide a list of certificate filenames in PEM format, and
```

Comment:
nit: "certificate filenames in PEM format" reads a little wonky to me. How about: "Provide a list of filenames for certificates in PEM format,"